<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Light</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: #196D94;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="capturaProcesos">
       <property name="font">
        <font>
         <family>Ubuntu Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="programName">
          <property name="font">
           <font>
            <family>Terminal</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>SIMULADOR DE PROCESOS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="processAcountLB">
              <property name="minimumSize">
               <size>
                <width>96</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>96</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Ubuntu Light</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>No. Procesos:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="processNumberSP">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Ubuntu Light</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QPushButton" name="processAcountPB">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(191, 191, 191);</string>
              </property>
              <property name="text">
               <string>ACEPTAR</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mostrarProcesos">
       <property name="maximumSize">
        <size>
         <width>1270</width>
         <height>720</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="pendientLotsLB">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>No. Lotes pendientes: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="pendientLotsLCD"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="lotWorkingGB">
          <property name="maximumSize">
           <size>
            <width>255</width>
            <height>221</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Light</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Lote trabajando</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableWidget" name="lotWorkingTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>32</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>70</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>TME</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>TT</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="processRuningGB">
          <property name="maximumSize">
           <size>
            <width>255</width>
            <height>221</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Proceso en ejecuci√≥n</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QTableWidget" name="processRuningTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;
</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>36</number>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string>ID</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>OPE</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>TME</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>TT</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>TR</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="finishedGB">
          <property name="maximumSize">
           <size>
            <width>430</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Terminados</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTableWidget" name="finishedTB">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color: black;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>OPE</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>RL</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>NL</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="globalCountLB">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Contador global: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="globalCountLCD"/>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="startRunPB">
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: black;
background-color: white;</string>
          </property>
          <property name="text">
           <string>Comenzar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
